static/app/components/events/interfaces/spans/aggregateSpanDetail.tsx

```diff
@@ -16,7 +16,7 @@ import type {
 } from 'sentry/utils/performance/quickTrace/types';
 import {useLocation} from 'sentry/utils/useLocation';
 import useProjects from 'sentry/utils/useProjects';
-import {TraceViewSources} from 'sentry/views/performance/newTraceDetails/traceMetadataHeader';
+import {TraceViewSources} from 'sentry/views/performance/newTraceDetails/traceHeader/breadcrumbs';
```

Comment:
The large numbe of files impacted by this PR is due to moving the `TraceViewSources` enum to the new `traceHeader > breadcrumbs.tsx` file.